docs/introduction/en/basics.md

```diff
@@ -18,6 +18,21 @@ To incentivize the maintenance of the Conflux network and charge users for consu
 | 10^12         | uCFX          |
 | 10^18         | Conflux(CFX)  |
 
+As a native token of the Conflux network, CFX plays a very important role in the stability of the system.
+
+First of all, CFX is used to pay transaction fees and to store collateral. Transactions are packaged and executed by miners. In order to incentivize miners to keep participating in the bookkeeping and provide sufficient security for the network, each transaction is charged a (gas) fee calculated as gasPrice * max(gasLimit * 3/4, gasUsed), which is paid by CFX. In addition, if the execution of a transaction takes up new storage space, a portion of the CFX is pledged for the use of the storage space (the pledged CFX can be released under certain condition).
+
+Conflux introduces a [staking mechanism (Protocol Specification chapter 8.3)](https://confluxnetwork.org/files/Conflux_Protocol_Specification_20201020.pdf). This mechanism not only provides a better way to pay for storage usage (as opposed to "pay once, occupy forever") but also serves to construct the voting rights of Conflux decentralized governance. The annualized income of collateralized CFX is around 4%. 
```

Comment:
This part is confusing, it mixes staking through internal contract and storage collateral together.